cypress/utils/apps/query_enhancements/constants.js

```diff
@@ -7,6 +7,7 @@ export const DATASOURCE_NAME = 'data-logs-1';
 export const WORKSPACE_NAME = 'query-ws';
 export const START_TIME = 'Jan 1, 2020 @ 00:00:00.000';
 export const END_TIME = 'Jan 1, 2024 @ 00:00:00.000';
+export const SAVE_QUERY_CONFLICT_NAME_ERROR_TEXT = 'Name conflicts with an existing saved query';
```

Comment:
nitpick: i feel like this should belong in `saved_queries.js`. Yes it is a constant, but its a constant thats only used by `saved_queries`, so other enhancement tests don't need this

Reply:
Very fair comment. Moving.